src/kosha/app/util.clj

```diff
@@ -1,12 +1,12 @@
 (ns kosha.app.util
   (:require [cheshire.core :as json]
-            [immuconf.config :as cfg]))
+            [immuconf.config :as config]))
 
 (defn json-response [response]
   {:status 200
    :body (json/encode response)
    :content-type "application/json"})
 
-(defonce config (cfg/load "resources/config.edn"))
-(defn get-config [& keys]
-  (apply (partial cfg/get config) keys))
+(defonce config (config/load "resources/config.edn"))
```

Comment:
ah, I now see why you'd chosen `cfg`, it is shadowed here. Maybe use `config-map`, or `config-store` here?